main/freetype/APKBUILD

```diff
@@ -22,11 +22,6 @@ source="http://download.savannah.gnu.org/releases/freetype/freetype-$pkgver.tar.
 	freetype-profile.sh
 	"
 
-# secfixes:
-#   2.7.1-r1:
-#     - CVE-2017-8105
-#     - CVE-2017-8287
-
```

Comment:
I think they only make the APKBUILD messier.
Isn't the place of that comments on alpine-secdb?
I think it fits better there.
But if you want I can re-add those comments here, I just don't like it personally.